examples/flutter_gallery/lib/demo/shrine/shrine_page.dart

```diff
@@ -88,6 +88,8 @@ class ShrinePageState extends State<ShrinePage> {
       appBar: new AppBar(
         elevation: _appBarElevation,
         backgroundColor: Theme.of(context).cardColor,
+        iconTheme: IconTheme.of(context),
```

Comment:
I think the code is correct although I could have used `Theme.of(context).iconTheme` if I'd set up the app's overall Theme that way.  

This stuff has become complicated. The Icon class says that its color comes from the current `IconTheme`. The DartDoc for `IconTheme.of()` says that its value defaults to `Theme.of(context).iconTheme`.  So if you're an app developer, which inherited widget do you want? 

And AppBar's iconTheme defaults to `Theme.of(context).primaryIconTheme`.